packages/components/src/gradient-picker/index.tsx

```diff
@@ -213,6 +213,7 @@ export function GradientPicker( {
 	onChange,
 	value,
 	clearable = true,
+	disableAlpha = false,
```

Comment:
It will make sense to add the new prop to the `GradientPicker` README.

Reply:
I'll try to get #67250 merged first so we don't need to do this manually 🙂

Reply:
Done. @wwdes If you rebase on or merge back `trunk`, running `npm run docs:components` should automatically update the README according to the TypeScript data.